salt/states/npm.py

```diff
@@ -274,6 +274,17 @@ def bootstrap(name,
     '''
     ret = {'name': name, 'result': None, 'comment': '', 'changes': {}}
 
+    if __opts__['test']:
+        try:
+            call = __salt__['npm.install'](dir=name, runas=user, pkg=None, dry_run=True)
+        except (CommandNotFoundError, CommandExecutionError) as err:
+            ret['result'] = False
+            ret['comment'] = 'Error Bootstrapping {0!r}: {1}'.format(name, err)
+            return ret
+        ret['result'] = None
+        ret['comment'] = 'Directory was successfully bootstrapped'
```

Comment:
Is it possible to get the results of `call` and put that information in the `changes` dictionary? If we're returning `None` here because changes will occur, we should specify what those changes will be. I think updating the comment to reflect that will be a nice change as well.